icon_search.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
To be consistent with the other scripts this should be 
```suggestion
#!/usr/bin/env bash
```